source/installation/kafka-deployments/migrator-with-kafka.txt

```diff
@@ -86,7 +86,7 @@ Learn More
    :hidden:
    :titlesonly:
 
-   /installation/kafka-deployments/install-kafka-cluster
-   /installation/kafka-deployments/install-docker
-   /installation/kafka-deployments/install-confluent-kafka
-   /installation/kafka-deployments/advanced-settings
+   Use Existing Kafka Cluster </installation/kafka-deployments/install-kafka-cluster>
+   Use a Containerized Cluster with Docker </installation/kafka-deployments/install-docker>
```

Comment:
[Suggestion]: Drop the indefinite article.

Reply:
Done